activestorage/test/service/disk_service_test.rb

```diff
@@ -3,16 +3,16 @@
 require "service/shared_service_tests"
 
 class ActiveStorage::Service::DiskServiceTest < ActiveSupport::TestCase
-  tmp_config = { tmp: { service: "Disk", root: File.join(Dir.tmpdir, "active_storage") } }
-  SERVICE = ActiveStorage::Service.configure(:tmp, tmp_config)
+  TMP_CONFIG = { tmp: { service: "Disk", root: File.join(Dir.tmpdir, "active_storage") } }
+  SERVICE = ActiveStorage::Service.configure(:tmp, TMP_CONFIG)
 
   include ActiveStorage::Service::SharedServiceTests
 
-  test "name" do
+  def test_name
```

Comment:
I think we prefer test blocks for newer tests. Is there a reason why you chose to change this?

Reply:
https://github.com/rails/rails/pull/42163#discussion_r627749039